data/json/npcs/TALK_ALLY_TUTORIAL.json

```diff
@@ -100,7 +110,7 @@
   {
     "id": "TALK_ALLY_TUTORIAL_ACTIVITIES",
     "type": "talk_topic",
-    "dynamic_line": "I can help with some tasks if you show me where to work.\n  Use the zone manager (keybind 'Y') to set up sorting zones for your loot, or to draw blueprints for a building, or to define where  you want to plant some crops, or where you'd like some trees cut down, or where you want a vehicle dismantled.  Then tell me to sort stuff, or build stuff, or cut down trees, or dismantle a vehicle or do farmwork, and I'll go off and do my best to get what you want done.  If I need tools, you should leave them pretty close to where you want me to work - axes for logging, shovels and seeds and fertilizer for farming, wrenches and hacksaws or a toolbox to take apart a vehicle.  I can pretty much sort out our stuff without needing tools, but keep the piles of unsorted and sorted stuff kind of close together because I don't want to walk back and forth carrying junk too much.",
+    "dynamic_line": "I can help with some tasks if you show me where to work.\n  Use the zone manager (keybind 'Y') to set up sorting zones for your loot, or to draw blueprints for a building, or to define where  you want to plant some crops, or where you'd like some trees cut down, or where you want a vehicle dismantled or repaired, or a good fishing spot.  Then talk to me about my current activity and tell me to sort stuff, or build stuff, or cut down trees, or repair or dismantle a vehicle,  or do farmwork, or catch some fish, and I'll go off and do my best to get what you want done.  If I need tools, you should leave them pretty close to where you want me to work - axes for logging, shovels and seeds and fertilizer for farming, wrenches and hacksaws or a toolbox to take apart a vehicle.  I can pretty much sort out our stuff without needing tools, but keep the piles of unsorted and sorted stuff kind of close together because I don't want to walk back and forth carrying junk too much.",
```

Comment:
May want to clarify here, that if the activity requires a tool in hand / invntory, the tool needs to be in a loot spot ( any loot spot will do )
Although if its something like construction or vehicle, then they will move tools and components near the work spot, or use them if they already near, thats the distinction.

And you may wnt to clarify that they will tidy up after the activity, putting unused components, and used tools, and activity products ( i.e fish from fishing ) into the nearest unsorted loot zone

Reply:
Thanks for the details.  Fixed.